packages/suite/src/components/suite/Translation/components/BaseTranslation/index.tsx

```diff
@@ -1,6 +1,7 @@
 import React from 'react';
-import { FormattedMessage, MessageDescriptor } from 'react-intl';
-import messages from '@suite/support/messages';
+import { FormattedMessage } from 'react-intl';
+import messages from '../../../../../support/messages';
```

Comment:
we cannot use aliases in this file because of temporary hack `@suite-common/intl-types` until #5647 will be merged